<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Piero Proietti</author>
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogDad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pengui/icons/pengui.svg</normaloff>:/pengui/icons/pengui.svg</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMakeIsohybrid">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>260</y>
     <width>122</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create isohybrid ISO image</string>
   </property>
   <property name="text">
    <string>isohybrid</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMakeMd5sum">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>300</y>
     <width>245</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create MD5/SHA256 Checksum of the live ISO</string>
   </property>
   <property name="text">
    <string>md5sum</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>247</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>16</y>
     <width>771</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_snapshot_dir">
      <property name="toolTip">
       <string>The place were we make eggs, usually /home/eggs</string>
      </property>
      <property name="text">
       <string>Nest</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditSnapshotDir">
      <property name="toolTip">
       <string>The place were we make eggs, usually /home/eggs</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_snashot_prefix">
      <property name="toolTip">
       <string>Prefix to be used with basenage to get ISO image filename</string>
      </property>
      <property name="text">
       <string>Prefix</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditSnapshotPrefix">
      <property name="toolTip">
       <string>Prefix to be used with basenage to get ISO image filename</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_snapshot_basename">
      <property name="toolTip">
       <string>volid of the ISO image</string>
      </property>
      <property name="text">
       <string>Basename</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditSnapshotBasename">
      <property name="toolTip">
       <string>volid of the ISO image</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_user_opt">
      <property name="toolTip">
       <string>Live user name for live system/ISO image</string>
      </property>
      <property name="text">
       <string>Live user name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditUserOpt">
      <property name="toolTip">
       <string>Live user name for live system/ISO image</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_user_opt_passwd">
      <property name="toolTip">
       <string>Live user password for live system/ISO image</string>
      </property>
      <property name="text">
       <string>Live user password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditUserOptPasswd">
      <property name="toolTip">
       <string>Live user password for live system/ISO image</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_root_passwd">
      <property name="toolTip">
       <string>root password for live system/ISO image</string>
      </property>
      <property name="text">
       <string>Live root password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditRootPasswd">
      <property name="toolTip">
       <string>root password for live system/ISO image</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
